specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/softwareUpdateDeployment.json

```diff
@@ -0,0 +1,628 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Compute Management- Update Management API",
+		"description": "APIs for managing software update configurations.",
+		"version": "2016-04-30-preview"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security": [{
+		"azure_auth": [
+			"user_impersonation"
+		]
+	}],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/SoftwareUpdateDeployments/{softwareUpdateDeploymentName}": {
```

Comment:
Is the leading 'S' in `SoftwareUpdateDeployments` supposed to be capitalized?  I believe this is the reason for [this](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/462070669) failure.